### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by C-Hemanth-kumar-Reddy** [microsoft/ML-For-Beginners#698] [Q&A] [answered]
What is open source project

**Comment by Slotter881** [comment, marked as answer]
e[@@](https://github.com/danyadev/vk-desktop/releases/download/0.3.3/VK.Desktop.mac.arm64.dmg)__

**Comment by XavierRolex** [comment]
An open source project is a software project whose source code is freely available for anyone to view, use, modify, and distribute. This is typically done under an open source license, such as MIT, Apache 2.0, or GPL.

**Comment by Hamza-farid** [comment]
An open-source project is a project where the source code is made publicly available so that anyone can:

View how it works

Use and modify it for their own purposes

Contribute back by fixing bugs, adding features, or improving docs

Most open-source projects are shared on platforms like GitHub, under a license (like MIT, Apache, GPL) that explains what you’re allowed to do.

Example: This repo itself is open-source  you can read the code, suggest changes, and even reuse parts of it in your own projects (following the license).

**Comment by Shaikhasna** [comment]
An open-source project is a software project where the source code is publicly available, so anyone can view, use, modify, and contribute. Open-source encourages learning, collaboration, and sharing improvements with the community. Most projects have a license (like MIT, Apache, or GPL) that explains what you can do with the code.

Example: This repo itself is open-source — you can explore the code, suggest improvements, or even use parts of it in your own projects while following the license.

**Reply by C-Hemanth-kumar-Reddy** [reply]
Is python basics okay or need a deep coding skills , as what ever the open source proj to see, everything looks like high level
